lib/mock-directive/mock-directive.spec.ts

```diff
@@ -142,6 +151,12 @@ describe('MockDirective', () => {
     expect(debugElement.nativeElement.innerHTML).toContain('hi');
   });
 
+  it('displays structural directive content by default when arg is true', () => {
+    const debugElement = fixture.debugElement.query(By.css('#another-example-structural-directive'));
```

Comment:
One more thing here is ngIf, should we render it or not by default? I would say depends, and the best option would be its default behaviour and that's why CommonModule isn't mocked, or - depends on developer, which behaviour of the code he wants to test.

Reply:
Oh, I see...that is a bummer. I have a few cases where I have structural directives that are basically helper directives, like for feature toggles, where they are like prewired ngIfs that take an arg. Typically for these I just want them render by default because I don't care to test that the structural directive works but I do want to test the content.

As far as the ngIf, I feel like it would be a smell to mock that in the first place.

Reply:
I guess, but not sure, there's a way to detect if child elements use context variables, then we would solve the issue.